File: Java_Advanced/gsapi/src/main/java/br/com/fiap/gs/gsapi/dto/external/GoogleGeocodingGeometryDTO.java
package br.com.fiap.gs.gsapi.dto.external;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class GoogleGeocodingGeometryDTO {

    @JsonProperty("location")
    private GoogleGeocodingLocationDTO location;

    public GoogleGeocodingGeometryDTO() {
    }

    // Getters e Setters
    public GoogleGeocodingLocationDTO getLocation() {
        return location;
    }
    public void setLocation(GoogleGeocodingLocationDTO location) {
        this.location = location;
    }
}

File: Advanced_Business_Development_with.NET/gsApi/gsApi/dto/response/GoogleGeocodingGeometryDto.cs
// File: gsApi/dto/response/GoogleGeocodingGeometryDto.cs
using System.Text.Json.Serialization;
namespace gsApi.dto.response
{
    public class GoogleGeocodingGeometryDto
    {
        [JsonPropertyName("location")]
        public GoogleGeocodingLocationDto? Location { get; set; } // Anulável
    }
}